fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStore.java

```diff
@@ -350,6 +353,15 @@ public RecordMetaDataProvider getMetaDataProvider() {
         return metaDataProvider;
     }
 
+    /**
+     * Get the provider for the record store's meta-data.
```

Comment:
It looks like this description needs to be updated after a copy/paste error

Reply:
done